cg/meta/demultiplex/demux_post_processing.py

```diff
@@ -120,20 +117,75 @@ def finish_flow_cell_temp(self, flow_cell_name: str) -> None:
         if not parsed_flow_cell:
             return
 
+        # 3. Create and store flow cell in status db.
         flow_cell: Flowcell = self.create_flow_cell(parsed_flow_cell=parsed_flow_cell)
-
-        # 4. Store flow cell in status db.
         self.status_db.session.add(flow_cell)
         self.status_db.session.commit()
 
-        # 5. Store flow cell data in housekeeper.
+        # 4. Store flow cell data in housekeeper.
         self.add_flow_cell_data_to_housekeeper(
             flow_cell_name=flow_cell_name, flow_cell_directory=flow_cell_dir
         )
 
-        # 6. Create sequencing metrics
+        # 5. Create sequencing metrics.
         self.add_sample_lane_sequencing_metrics_for_flow_cell(flow_cell_name=flow_cell_name)
 
+        # 6. Update samples in status db with read counts and sequencing date.
+        sample_ids: List[str] = self.get_sample_ids_from_sample_sheet(
+            parsed_flow_cell_directory=parsed_flow_cell
+        )
+        flow_cell_sequencing_date: datetime = parsed_flow_cell.run_date
+
+        self.update_samples_with_read_counts_and_sequencing_date(
+            sample_internal_ids=sample_ids, flow_cell_sequencing_date=flow_cell_sequencing_date
```

Comment:
I think we talked about this. I am unsre whether a "sample" sequencing date makes sense to have. 
Samples run on one or multiple flow cells, depending whether they need top ups etc. the samplelanesequencing metrics will have this link already. So this information will be 
1. inaccurate in cases of multiple runs 
2. redundant since it will be present in another table. 

what about 
1. renaming it to last_sequenced_at 
2. removing it and have the information in the flow cell table which can be easily found with the new sample lane sequencing metrics link

@henrikstranneheim, @Vince-janv, @diitaz93 , @islean

Reply:
>     2. removing it and have the information in the flow cell table which can be easily found with the new sample lane sequencing metrics link
> 
> 
Sound lime a good suggestion - keep it DRY

Reply:
Ok! I kept the date update for the sample since that is how it is done currently. But I will remove the update of it here and then we can remove the `sequenced_at` from the sample model in a separate PR.